src/test/resources/bootstrap/fall-out-boy.json

```diff
@@ -10,7 +10,35 @@
 					"artworkUrl": "https://i1.sndcdn.com/artworks-000057992801-cxjxj7-large.jpg",
 					"soundCloudId": 202330997,
 					"duration": 300000
-				}
+				},
+			  {
+				"id": 2,
+				"name": "Thanks for the Memories",
```

Comment:
We might want to change the data here so we can actually have meaningful songs when running locally.